doas.c

```diff
@@ -188,7 +263,13 @@ checkconfig(const char *confpath, int argc, char **argv,
 	if (setresuid(uid, uid, uid) != 0)
 		err(1, "setresuid");
 
+#ifdef DOAS_CONFDIR
+	if (isconfdir(confpath))
+		parseconfdir(confpath, 0);
+	else
+#else
 	parseconfig(confpath, 0);
+#endif
```

Comment:
This is wrong. If `DOAS_CONFDIR` is defined, the result after preprocessing will be:

```c
	if (isconfdir(confpath))
		parseconfdir(confpath, 0);
	else
	if (!argc)
		exit(0);
```

The same on line 414.

Reply:
Yep, my bad, I forgot to take the #else out.